lib/generators/shopify_app/shopify_app_generator.rb

```diff
@@ -2,8 +2,13 @@ module ShopifyApp
   module Generators
     class ShopifyAppGenerator < Rails::Generators::Base
 
+      def initialize(args, *options)
+        @opts = options.first
```

Comment:
I didn't know about that but I am not sure it helps me here. I am just grabbing what I need here and sending downwards - here is where it actually gets used https://github.com/Shopify/shopify_app/blob/master/lib/generators/shopify_app/install/install_generator.rb#L12